<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>串口A设置：</string>
         </property>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>205</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>串口号：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cmbCOMA"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cmbBaudRateA"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>奇偶校验：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cmbParityA"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>数据位：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cmbDataBitsA"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cmbStopBitsA"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>串口B设置：</string>
         </property>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>205</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>串口号：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cmbCOMB"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cmbBaudRateB"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>奇偶校验：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cmbParityB"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>数据位：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cmbDataBitsB"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cmbStopBitsB"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>状态：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <widget class="QTextBrowser" name="txtOutput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>53</y>
          <width>441</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>输出：</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>是否监听消息（可能会影响效率）</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnClear">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>30</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>开始转发</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
